xhr/sync-no-progress.any.js

```diff
@@ -0,0 +1,6 @@
+test(t => {
+  let xhr = new XMLHttpRequest();
+  xhr.onprogress = t.unreached_func('progress event should not be fired');
+  xhr.open('GET', 'resources/trickle.py?count=4&delay=150');
```

Comment:
Okay, this is why we have code reviews :-)

The test didn't fail, because it ended before onprogress is called.

I've fixed it, and also added a check that the "load" event *is* called to reassure myself that it's actually testing something now.